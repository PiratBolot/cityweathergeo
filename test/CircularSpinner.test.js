import React from 'react';
import { shallow } from '../enzyme/enzyme';
import { shallowToJson } from 'enzyme-to-json';

import CircularSpinner from "../src/components/preLoader/spinner/CircularSpinner";

describe('Circular Spinner', () => {
    it('should render correctly', () => {
        const output = shallow(
            <CircularSpinner />
        );
        expect(shallowToJson(output)).toMatchSnapshot();
    });
});