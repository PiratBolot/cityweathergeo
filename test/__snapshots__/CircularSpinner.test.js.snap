// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Circular Spinner should render correctly 1`] = `
<div
  className="loader"
>
  <svg
    className="circular"
    viewBox="25 25 50 50"
  >
    <circle
      className="path"
      cx="50"
      cy="50"
      fill="none"
      r="20"
      strokeMiterlimit="10"
      strokeWidth="2"
    />
  </svg>
</div>
`;
